src/main/java/net/minecraftforge/gradle/user/patcherUser/TaskApplyBinPatches.java

```diff
@@ -106,7 +106,6 @@ public void doTask() throws IOException
                 {
                     ZipEntry n = new ZipEntry(e.getName());
                     n.setTime(e.getTime());
-                    out.putNextEntry(n);
```

Comment:
@mezz I think this might be the problem.

Not sure why this line is removed - I don't remember wanting to remove it, but obviously I did... :(

Reply:
Ah... that would do it heh